.scalafmt.conf

```diff
@@ -1,6 +1,8 @@
-# Matches the version here
-# https://github.com/wellcomecollection/dockerfiles/blob/master/scalafmt/Dockerfile#L11
-version = "2.0.0"
+version = "3.7.1"
 align.openParenCallSite = false
 continuationIndent.defnSite = 2
 rewrite.rules = [SortImports]
+runner.dialect = scala212
+
+newlines.implicitParamListModifierPrefer = before
+newlines.beforeCurlyLambdaParams = multiline
```

Comment:
I suspect that the behaviour behind both of these settings are new.  I chose these, because they seemed to match the more frequent use.  Leaving these unset led to more changes (and ones that I personally thought were a bit ugly)